packages/react/accordion/src/Accordion.test.tsx

```diff
@@ -60,44 +61,164 @@ describe('given a basic Accordion', () => {
   });
 
   describe('when clicking a button', () => {
+    let button: HTMLElement;
+    let panelOne: HTMLElement | null;
+    let panelTwo: HTMLElement | null;
+
     beforeEach(async () => {
       button = rendered.getByText('Button One');
       fireEvent.click(button);
       await waitFor(() => {
-        panel = rendered.getByText('Panel One');
-        expect(panel).toBeVisible();
+        panelOne = rendered.getByText('Panel One');
+        return panelOne;
```

Comment:
what does it do to return it here? Is that necessary considering it's not assigned?

Reply:
I thought `waitFor` expects a promise returned to is knows what to wait for... could be wrong

Reply:
Yeah, `return`s are completely redundant... removed!